### ISSUE TITLE
Sockets CLI

### ISSUE THREAD

**Issue by crookse** [drashland/wocket#23]
## Summary

What:

Make a Sockets CLI via `sockets` command.

Why:

It'd be nice to have a CLI that users can use to scaffold a Sockets application.

## Acceptance Criteria
- [ ] Use [drashland/services/cli](https://github.com/drashland/services/tree/master/cli) to help build out CLI
- [ ] Write unit tests
- [ ] Write integration tests
- [ ] Write documentation (https://github.com/drashland/website/issues/249)
- [ ] Add subcommand: `make`
    - This subcommand should be able to scaffold an application
    - Usage would be `sockets make name_of_application`

## Taking on this issue? Read this before you get started.

If you take on this issue and the acceptance criteria does not make sense, post comments on this issue with your questions. Also, you can always hop in the Discord channel at https://discord.gg/c5R3sYuR to get help quickly.

Please read the [Contributing Guidelines](https://github.com/drashland/.github/blob/master/CONTRIBUTING.md) before starting this issue.

**Comment by saragee3** [comment]
what is the point of this? Wouldn't this `make` command be a mirror the small `Quick Start` section? IMO, this is overkill and too much control over other dev envs. It _does_ make sense in Drash S though if the user opts in.

**Comment by crookse** [comment]
agreed. we can revisit if someone comes to us with a use case
